pkg/apis/configuration/validation/transportserver.go

```diff
@@ -249,8 +249,8 @@ func validateHashLoadBalancingMethod(method string, fieldPath *field.Path, isPlu
 		allErrs = append(allErrs, varErrs...)
 	}
 
-	if !escapedStringsFmtRegexp.MatchString(method) {
-		msg := fmt.Sprintf("invalid value for hash: %v", validation.RegexError(escapedStringsErrMsg, escapedStringsFmt))
+	if err := validateEscapedString(method); err != nil {
+		msg := fmt.Sprintf("invalid value for hash: %v",err.Error())
```

Comment:
there is no need to call Error() here
```suggestion
		msg := fmt.Sprintf("invalid value for hash: %v", err)
```